Sources/StreamChat/Models/Message.swift

```diff
@@ -225,7 +220,7 @@ public extension _ChatMessage {
     func attachments<Payload: AttachmentPayload>(
         payloadType: Payload.Type
     ) -> [_ChatMessageAttachment<Payload>] {
-        attachments.compactMap {
+        _attachments.compactMap {
             $0.attachment(payloadType: payloadType)
         }
```

Comment:
I'm not sure about this decision. If we expose it only like this it's not possible to check the total amount of attachments of the message because you will always get just the attachment of the type you specify.

Reply:
Reverted those changes and kept attachments publicly exposed. Will change this in the scope of `attachment score` PR